### DISCUSSION TITLE
About the file "ReadMe_Overview.txt" content

### DISCUSSION THREAD

**Discussion by huangjc0715** [apprenticeharper/DeDRM_tools#1762] [Q&A]
There's a line in the "ReadMe_Overview.txt": 
DeDRM application for Mac OS X users: (Mac OS X 10.6 and above)
---------------------------------------------------------------
DeDRM application for Windows users: (Windows XP through Windows 10)
------------------------------------------------------------------
As of Version 6.7 of the tools, these are no longer provided or supported.

I can not understand，is it means that  DeDRM_tools can not delete DRM in calibre-5.x?Can this function still be used in calibre-5.x?

emmm.....I con't speak English very well and I'm learning it，I hope you see what I mean.

**Comment by ElleKayEm** [comment]
Those were DeDRM programs that could be used on their own without calibre.  They are no longer included.  The DeDRM calibre plugin version 7.2.1 can be used with calibre 5.x.

**Reply by huangjc0715** [reply]
Thank you! I got it!
